docs/OWNERS

```diff
@@ -0,0 +1,2 @@
+labels:
+- kind/documentation
```

Comment:
uhm, I'm not sure about this.
Any change that includes some docs will be tagged as kind/documentation, so the kind label check will pass and we will forget to set the right label - so in our release notes we'll have a lot of docs changes :D

Reply:
ah, good point, I can remove that for new indeed :upside_down_face: